packages/block-editor/src/store/reducer.js

```diff
@@ -1671,4 +1683,5 @@ export default combineReducers( {
 	hasBlockMovingClientId,
 	automaticChangeStatus,
 	highlightedBlock,
+	hoveredBlocks,
```

Comment:
I wonder if this should be added under the `blocks` reducers. Maybe not since it isn't crucial for blocks 🤔

Reply:
Im not certain, but I lean on the "maybe not" as well since it isn't crucial and is more experimental.  🤷‍♀️